### DISCUSSION TITLE
Add support to change font color when using colorScheme

### DISCUSSION THREAD

**Discussion by dailydm** [chakra-ui/chakra-ui#7228] [Ideas]
I have added a palette of custom colors that fit within the branding guidelines of my company. I have been loving the `colorScheme` prop for the Button component, but I need to change the color of the font for a few colorSchemes from "white" in dark mode to "black". 

When I did some digging into how the yellow, gray, and cyan colorSchemes are working in the default theme, I noticed that they are hardcoded in an "accessibleColorMap": https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/components/button.ts#L114

I would love if there would be a way to extend this accessibleColorMap to include some one off configurations for colors where the built in functionality doesn't work quite how I intend.

**Comment by AnonymousXC** [comment]
You can use semanticTokens.
https://youtube.com/watch?v=epJuxo8FKFA
U can check this out👆
